### PR TITLE
Update intro and readme text

### PR DESCRIPTION
**Pull request by tmieslinger**
* I suggest an update of the `README.md` moving some notes on myst and building the book to the file `CONTRIBUTING.md`. 
* I added some more information to the introduction to the book
* a major change that I would like to suggest is to get more precise on the code examples and rephrase them to `HALO` as we only have HALO code examples so par. That way we and others can easily add further scripts, while keeping an overview. What do you think about that?

### CODE REVIEW

File: how_to_eurec4a/halo.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# HALO
+
+The German High Altitude and Long-Range Research Aircraft (HALO) operated by the Deutsches Zentrum für Luft- and Raumfahrt (DLR) was equiped with remote sensing instruments and a facility to launch dropsondes. All instruments and PIs are listen on the official [eurec4a](http://eurec4a.eu/platforms/halo) webpage.
+
+
+During EUREC4A HALO flew most of the time at about 9 km altitude circular patterns with 200 km diameter centered at 57.72W, 13.30N. 
+On typical flight days an excursion to the NTAS buoy or a satellite underpass separated the first three from the remaining circles. Typically, the aircraft stayed up in the air for 8-9 hours per day starting at varying times in the morning.
```
Comment (JuleRadtke) [inline how_to_eurec4a/halo.md:7]: The typical number of circles was six right? So probably add to get this number in: "On typical flight days an excursion ...  separated the first three from the remaining three circles."?
And for the varying starting times, probably add the explanation: "... starting at varying times in the morning to capture the diurnal cycle."

**Review by JuleRadtke — APPROVED** [review]
Great!! I like the restructuring, good Idea! I only have some tiny comments.

File: how_to_eurec4a/halo.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# HALO
+
+The German High Altitude and Long-Range Research Aircraft (HALO) operated by the Deutsches Zentrum für Luft- and Raumfahrt (DLR) was equiped with remote sensing instruments and a facility to launch dropsondes. All instruments and PIs are listen on the official [eurec4a](http://eurec4a.eu/platforms/halo) webpage.
```
[the remark this answers is not in the document]
Comment (tmieslinger) [reply how_to_eurec4a/halo.md:3]: up, thanks.

File: how_to_eurec4a/halo.md (hunk 1 above)
Comment (tmieslinger) [reply how_to_eurec4a/halo.md:7]: good point.

**Comment by MeraX** [comment]
I wonder if it would make sense to highlight more of the "custom requrements" compared to general stuff like numy, matplotlib and xarray. 

At least I would like to install just those packages that are necessary to execute some of the examples in my environment. I had the basics installed, but was for example wondering about the `eurec4a` module. (Note, it is available on [PyPi](https://pypi.org/project/eurec4a/) and [GH](https://github.com/eurec4a/pyeurec4a/).

Therefore, I would welcome if you/we would highlight those packages that were built for EUREC4A and this how_to_eurec4a. I think it is especially the `eurec4a` package. Perhaps also `intake`, `intake-xarray`, and `ipfsspec>=0.0.2'  (?).

I'm always a bit hesitant to install a lot of random packages without understanding their need.
Would it make sense to highlight all packagees required in each notebook at its beginning?

And are actually all requirements listed in requirements.txt directly necessary or are some of them only required by e.g. `eurec4a` and thus also listed in `eurec4a`'s requirements?

**Comment by d70-t** [comment]
I fixes the smaller things we've discussed previously and am currently in the process of adding a little "running locally" chapter which will be linked from the intro page. This should provide a little more explanation about the required things but as discussed in #27 the exact requirements for local environments might still change a bit in the next time.

**Comment by d70-t** [comment]
I've added a few commits. @tmieslinger, @MeraX are you happy with the situation? I'd be happy to merge it as is.

**Comment by MeraX** [comment]
Grounding but very happy. 

Your commits address several of my issues

**Pull request merged**